web/src/main/webapp/lamp.ts

```diff
@@ -295,12 +305,13 @@ export class LAMPDialog extends Dialog {
     constructor(private selectedColumns: string[], private page: FullPage,
                 private schema: Schema, private remoteObject: TableView) {
         super("LAMP");
-        this.addTextField("numSamples", "No. control points", "Integer", "15");
+        this.addTextField("numSamples", "No. control points", "Integer", "20");
         this.addSelectField("controlPointSelection", "Control point selection", ["Random samples", "Category centroids"], "Random samples");
         let catColumns = [""];
         for (let i = 0; i < schema.length; i++)
             if (schema[i].kind == "Category")
                 catColumns.push(schema[i].name);
+        this.addTextField("seed", "Random seed", "Integer", "1");
```

Comment:
I am not sure you really want the user to give you this. 
You can generate it locally somehow. The point is that we can re-execute the message later and it will generate the same result. The message will contain the seed.

Reply:
I will merge this, you can make the seed change later.